packages/asc-ui/src/components/Modal/Modal.tsx

```diff
@@ -36,28 +36,33 @@ const Modal: FunctionComponent<Props> = ({
 }) => {
   const ref = useRef<HTMLDivElement>(null)
   const { keyDown } = useTrappedFocus(ref)
+  const previouslyFocusedElement = document.activeElement
 
   useEffect(() => {
-    let renderedTimer = 0
-    const { current: node } = ref
-    if (node) {
-      clearTimeout(renderedTimer)
-      renderedTimer = setTimeout(() => {
-        node.focus()
-      })
-    }
-    return () => {
-      clearTimeout(renderedTimer)
+    if (open && ref.current) {
+      const node = ref.current
+      const firstFocusableEl = node.querySelector(FocusableElements.join(', '))
```

Comment:
Since the `node` variable is no longer used in callbacks we can instead use `ref.current` directly:
```suggestion
      const firstFocusableEl = ref.current.querySelector(FocusableElements.join(', '))
```